package.json

```diff
@@ -7,6 +7,7 @@
     "markdown-toc": "markdown-toc --bullets='-' -i"
   },
   "devDependencies": {
+    "@sigstore/cli": "^0.8.0",
```

Comment:
Your call but I would just pin to a specific version since renovate will update for us anyway. Provides a bit more consistent behavior as `npm install` won't try to use later versions and update the `package-lock.json` (and dirty a checkout). I don't really think we want to update deps outside of renovate PRs anyway.

Reply:
I think you're right. changed.